altair/examples/grouped_bar_chart_with_error_bars.py

```diff
@@ -0,0 +1,27 @@
+"""
+Grouped Bar Chart with Error Bars
+---------------------------
+This example shows a grouped bar chart with error bars.
+"""
+# category: bar charts
+import altair as alt
+from vega_datasets import data
+
+source = data.barley()
+
+bars = alt.Chart().mark_bar().encode(
+    x='year:O',
+    y=alt.Y('mean(yield):Q', title='Mean Yield'),
+    color='year:N',
+)
+
+error_bars = bars.mark_rule().encode(
```

Comment:
Since you're overwriting all encodings and marks in the bars object, there's no reason to build this of the ``bars`` object. Instead use ``alt.Chart().mark_rule()...``